client/signup/steps/domains/index.jsx

```diff
@@ -811,7 +811,12 @@ export class RenderDomainsStep extends Component {
 								) }
 							</div>
 						</div>
-						<Button primary className="domains__domain-cart-continue" onClick={ this.goToNext() }>
+						<Button
+							primary
+							className="domains__domain-cart-continue"
+							onClick={ this.goToNext }
```

Comment:
Changed this to only reference the function and not call it so we can unnest it.